packages/instant-apps-components/src/components/instant-apps-export/instant-apps-export.scss

```diff
@@ -91,3 +95,17 @@
     color: #6e6e6e;
   }
 }
+
+.hide {
+  display: none;
+}
+
+.screenshotCursor {
+  cursor: crosshair;
+}
+
+#maskDiv {
+  position: absolute;
+  background: red;
```

Comment:
down the line, it would be cool to have the mask background and border be configurable